tensorboard/plugins/profile/tf_op_profile/tf-op-profile.html

```diff
@@ -90,7 +94,8 @@ <h4>
         </p>
         <p>
           "Idle" represents the portion of the total execution time on device
-          that is idle.
+          that is idle. Wasted time is defined by RuntimeFraction * (1 -
```

Comment:
this looks pretty cody: how about using `<code>RuntimeFraction * (1 - max(...))</code>`?

Reply:
Done.